src/codeu/chat/database/DatabaseAccess.java

```diff
@@ -0,0 +1,34 @@
+package codeu.chat.database;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseCredentials;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import codeu.chat.util.Logger;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class DatabaseAccess {
+    private final String account = "/home/greg/Dropbox/gproj/codeu_project_2017/third_party/codeu-56fe6-firebase-adminsdk-6et4u-e78898c410.json";
```

Comment:
Remember to update the account path. Perhaps a relative rather than absolute path